### ISSUE TITLE
Gem installation error: uninitialized constant RailsSettingsUi::Types::Form

### ISSUE THREAD

**Issue by pkonovalov** [accessd/rails-settings-ui#58]
Caught gem installation error
```
 There was an error while trying to load the gem 'rails-settings-ui'. (Bundler::GemRequireError)
Gem Load Error is: uninitialized constant RailsSettingsUi::Types::Form
Backtrace for gem load error is:
/home/user/.rvm/gems/ruby-2.5.0@gemset/gems/rails-settings-ui-1.3.0/lib/rails-settings-ui/settings_form_coercible.rb:48:in `<class:SettingsFormCoercible>'
```

The problem in gem dry-types, which has recently updated to 0.13 version and renamed `Types::Form` to `Types::Params` (https://github.com/dry-rb/dry-types/blob/master/CHANGELOG.md). Need to lock rails-settings-ui to dry-types 0.12 or make it compatible with new version.

Temporary workaround is add to your Gemfile: `gem 'dry-types', '~> 0.12.2'`

**Comment by accessd** [comment]
fixed in 1.3.3 version

**Comment by chuchiperriman** [comment]
We're getting the same error in 1.3.3:


Gem Load Error is: uninitialized constant RailsSettingsUi::Types::Form
Backtrace for gem load error is:
/home/chuchi/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/rails-settings-ui-1.3.3/lib/rails-settings-ui/settings_form_coercible.rb:48:in `<class:SettingsFormCoercible>'

**Comment by edwardmp** [comment]
Fixed in #62
